2/contrib/jenkins/install-plugins.sh

```diff
@@ -367,10 +367,12 @@ function versionFromPlugin() {
 
 function installedPlugins() {
     echo "# Generated file: To generate this file run 'make plugins-list' before pushing your changes" > $BUNDLE_PLUGINS
+    temp_bundle=$(mktemp)
     for f in "$REF_DIR"/*.jpi; do
         echo "$(basename "$f" | sed -e 's/\.jpi//'):$(get_plugin_version "$f")"
-        echo "$(basename "$f" | sed -e 's/\.jpi//'):$(get_plugin_version "$f")" >> $BUNDLE_PLUGINS
+        echo "$(basename "$f" | sed -e 's/\.jpi//'):$(get_plugin_version "$f")" >> $temp_bundle
     done
+    sort $temp_bundle >> $BUNDLE_PLUGINS
```

Comment:
@akram given the discrepancy we are seeing in our local ubuntu / fedora systems, I want to confirm whether or not the supply chain check in this PR is in fact using you change here.

So add a `echo "generated plugin list $temp_bundle before sort"` followed by a `cat $temp_bundle` before you call to `sort` here.

Then after the call to `sort` add a `echo "generated plugin list $BUNDLE_PLUGINS after sort"` followed by a `cat $BUNDLE_PLUGINS` file

And then we can review the logs of the supply chain e2e

I am wondering if that e2e is *NOT* picking up these changes and is using the master branch copy, though maybe it is with the `clone: true` field you have at https://github.com/openshift/release/blob/master/ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml#L81-L82

Depending on the result, I'm wondering if we merge this now, then after this merges, we'll need a one time `make plugins-list` update to get the ordering correct

Reply:
done